finishline

```diff
@@ -16,6 +16,8 @@ import re
 import six
 import sys
 
+from mosaic import mosaic
```

Comment:
Ah, can you move this import inside a function or add a `try` `except ImportError` block around this?

Is mosaic on pypi?

Trying to figure out if adding it here will be a burden for other finishline users like @ashcrow.

Reply:
https://pypi.python.org/pypi/Mosaic ... not sure that's the same thing. It would end up causing an issue if it was expected to be present UNLESS it's installable via PyPi.